example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -325,125 +323,189 @@ describe('When there are granule differences and granule reconciliation is run',
     if (beforeAllFailed) fail('beforeAll() failed to prepare test suite');
   });
 
```

Comment:
L325-453 are just putting the existing tests to a suite.
Added "Internal Reconciliation Report" test suite.